v4m-chart/values.yaml

```diff
@@ -0,0 +1 @@
+# Default values for v4m
```

Comment:
This file is never actually used, is it?  If a user adds values here, are we using them in some way?  If has to be here or if we want it here b/c that is "best practices" for Helm charts...we should probably include comments indicating that there are no default values supported and the file isn't actually used.

Reply:
It's a convention that there is a default values.yaml file. We may use it in the future, but yes, for now, it's empty. I added a comment.